ELK/filebeat.yml

```
@@ -0,0 +1,18 @@
+filebeat.inputs:
+- type: log
+  enabled: true
+  paths:
+    - /app/logs/*.json
```

yamllint

[warning] wrong indentation: expected 2 but found 4

(indentation)